docs/howtoguides/how_to_know_if_system_affected_by_cve.rst

```diff
@@ -0,0 +1,185 @@
+    (1/3, 2/3) matanza, swish-e:
+    Ubuntu security engineers are investigating this issue.
+    (3/3) expat:
+    A fix is available in Ubuntu standard updates.
+    { apt update && apt install --only-upgrade -y expat }
+
+    2 packages are still affected: matanza, swish-e
+    ✘ CVE-2017-9233 is not resolved.
+
+CVE that requires a Pro service
+-------------------------------
+
+Assuming you have attached to a Pro subscription, when running the ``pro fix`` command
+you may see that the service required to fix the issue is not enabled. In this situation,
+``pro fix`` will also prompt you to enable that service.
+
+For example, if you want to fix the issue CVE-2023-1523 which requires the Pro service esm-infra 
+and you run the following command:
+
+.. code-block:: bash
+
+    $ sudo pro fix --dry-run CVE-2023-1523
+
+You will see the following output (if you type :kbd:`E` to enable the service when prompted):
+
+.. code-block:: text
+
+    CVE-2023-1523: snapd vulnerability
+     - https://ubuntu.com/security/CVE-2023-1523
+
+    1 affected source package is installed: snapd
+    (1/1) snapd:
+    A fix is available in Ubuntu Pro: ESM Infra.
+    The update is not installed because this system does not have
+    esm-infra enabled.
+
+    Choose: [E]nable esm-infra [C]ancel
+    > E
+    { pro enable esm-infra }
+    Updating Ubuntu Pro: ESM Infra package lists
+    Ubuntu Pro: ESM Infra enabled
+    { apt update && apt install --only-upgrade -y snapd ubuntu-core-launcher }
+
+    ✔ CVE-2023-1523 is resolved.
+
+We can observe that the required service was enabled and ``pro fix`` was able
+to successfully upgrade the affected package.
+
+Here we see how performing a dry run with the ``pro fix`` command is a quick and safe way to check if your system is affected by a specific CVE and also see if a fix is available.
+
+Success!
+========
+
+Congratulations! You have successfully determined if your system is affected by a specific CVE/USN using the `pro fix` command. If a fix is available, you can proceed to apply the fix by running the command provided in the output.
+To learn how to resolve a CVE using the `pro fix` command, refer to the guide on :ref:`How to resolve a CVE/USN? <pro-fix-resolve-cve>`.
+
+Additional Resources
+--------------------
+
+This is not the only scenario where you might want to use ``pro fix``. To find out about the other situations where it can be useful, as well as which options can be used to give you greater control over the command, you can refer to the following guides: 
+
+* In :ref:`Understanding scenarios encountered when using pro fix to solve a CVE/USN <pro-fix-howto>` you can continue learning more about the different scenarios you might encounter and understand the different outputs you will find.
+* :ref:`How do I know what the pro fix command would change? <pro-fix-dry-run>` will show you how to use ``pro fix`` in ``--dry-run`` mode to safely simulate the changes before they're applied.
```

Comment:
I'm now wondering - did we not already show how to do that in this page? Do we need to combine pages?

Reply:
I think so too, these two pages overlap in the information presented but the main difference I think from a user point of view is what the information presented represents. In one case it is for a user who is planning to use the `fix` command and wants to know what will change, on the other hand, the current added page let's the user check if the CVE affects their system or not.

It is basically the same thing but packaged differently for the ease of reading. let me know if you think we should actually just merge these into one. However, I think having these two guides as different pages answer different questions (I don't know how feasible it is to have both of them though}